ci-operator/step-registry/enable-qe-catalogsource/enable-qe-catalogsource-commands.sh

```diff
@@ -122,8 +122,9 @@ EOF
 function create_catalog_sources()
 {
     # get cluster Major.Minor version
-    ocp_version=$(oc version -o json | jq -r '.openshiftVersion' | cut -d '.' -f1,2)
-    index_image="quay.io/openshift-qe-optional-operators/aosqe-index:v${ocp_version}"
+    kube_major=$(oc version -o json |jq -r '.serverVersion.major')
+    kube_minor=$(oc version -o json |jq -r '.serverVersion.minor')
+    index_image="quay.io/openshift-qe-optional-operators/aosqe-index:v${kube_major}.${kube_minor}"
 
     echo "create QE catalogsource: qe-app-registry"
```

Comment:
Please also output the `index_image` here so that the user can check it easily, thanks!

Reply:
Done

Reply:
Print the index_image after line 129